experimental/virtio/src/console/mod.rs

```diff
@@ -148,38 +148,38 @@ where
     ) -> anyhow::Result<()> {
         self.device
             .start_init(DEVICE_ID as u32, inverse)
-            .map_err(|error| anyhow::anyhow!("Virtio error: {:?}", error))
-            .context("Couldn't initialize the PCI device.")?;
+            .map_err(|error| anyhow::anyhow!("virtio error: {:?}", error))
+            .context("couldn't initialize the PCI device")?;
         self.device
             .configure_queue(
                 RX_QUEUE_ID,
                 QUEUE_SIZE as u16,
                 translate(self.rx_queue.inner.get_desc_addr())
-                    .context("Failed to translate VirtAddr to PhysAddr")?,
+                    .context("failed to translate VirtAddr to PhysAddr")?,
```

Comment:
Maybe use `couldn't` here too?

Reply:
I was undecided about that, but will go ahead with it